### PR TITLE
buildchain: update K8s version to 1.12.10

### PR DESCRIPTION
**Pull request by NicolasT**

### CODE REVIEW

File: buildchain/buildchain/versions.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 # Project-wide versions {{{
 
 CALICO_VERSION     : str = '3.8.2'
-K8S_VERSION        : str = '1.12.9'
+K8S_VERSION        : str = '1.12.0'
```
Comment (slaperche-scality) [inline buildchain/buildchain/versions.py]: `0` instead of `10`?

Comment (NicolasT) [reply buildchain/buildchain/versions.py]: Whoops

**Comment by NicolasT** [comment]
@bert-e reset

**Review by slaperche-scality — APPROVED** [review]

**Pull request merged**